src/Microsoft.OpenApi.OData.Reader/Operation/EntitySetPostOperationHandler.cs

```diff
@@ -62,21 +62,42 @@ protected override void SetRequestBody(OpenApiOperation operation)
                 };
             }
 
-            // The requestBody field contains a Request Body Object for the request body
-            // that references the schema of the entity set’s entity type in the global schemas.
-            operation.RequestBody = new OpenApiRequestBody
+            IDictionary<string, OpenApiMediaType> content;
+
+            if (EntitySet.EntityType().HasStream)
             {
-                Required = true,
-                Description = "New entity",
-                Content = new Dictionary<string, OpenApiMediaType>
+                // Support creating a media entity
+                content = new Dictionary<string, OpenApiMediaType>
+                {
+                    {
+                        // TODO: Read the AcceptableMediaType annotation from model
```

Comment:
is an issue filed for this todo?

Reply:
Yes: https://microsoftgraph.visualstudio.com/Graph%20Developer%20Experiences/_workitems/edit/5854/